_notices/rsn0053.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: notice
+parent: RAPIDS Support Notices
+grand_parent: RAPIDS Notices
+nav_exclude: true
+notice_type: rsn
+# Update meta-data for notice
+notice_id: 53 # should match notice number
+notice_pin: true # set to true to pin to notice page
+title: "RAPIDS 25.10 consolidates Docker images to single tag per CUDA major version"
+notice_author: RAPIDS Ops
+notice_status: "In Progress"
+notice_status_color: yellow
+# 'notice_status' and 'notice_status_color' combinations:
+#   "Proposal" - "blue"
+#   "Completed" - "green"
+#   "Review" - "purple"
+#   "In Progress" - "yellow"
+#   "Closed" - "red"
+notice_topic: Platform Support Change
+notice_rapids_version: "v25.10"
+notice_created: 2025-09-24
+# 'notice_updated' should match 'notice_created' until an update is made
+notice_updated: 2025-09-24
+---
+
+## Overview
+
+Starting with RAPIDS `v25.10`, RAPIDS is consolidating Docker images to use a single rolling tag per CUDA major version instead of maintaining separate tags for each CUDA minor version. This change simplifies image selection and reduces maintenance overhead while maintaining compatibility within CUDA major versions.
+
+## Changes
+
+### New Tagging Strategy
+
+RAPIDS will now publish Docker images using the following naming convention:
+
+```
+<RAPIDS_VERSION>-cuda<CUDA_MAJOR_VERSION>-py<PYTHON_VERSION>
+```
+
+For example: `rapidsai/base:25.10-cuda12-py3.13` instead of `rapidsai/base:25.10-cuda12.9-py3.13`.
+
+### What This Means
+
+- **CUDA 12**: The `cuda12` tag will point to the latest supported minor version (currently 12.9)
+- **CUDA 13**: With the addition of CUDA 13, only `cuda13` tags will be published
```

Comment:
```suggestion
- **CUDA 12**: The `cuda12` tag will point to the latest supported minor version for that RAPIDS release.
- **CUDA 13**: Only `cuda13` tags will be published, and point to the latest supported minor version for that RAPIDS release.
```

Proposing:

* remove language about "currently", as that's at risk of becoming obsolete over time
* clarify that the `cuda13` images also are the latest supported minor version (this was not clear)
* clarify that these are **latest supported for that RAPIDS release**

Reply:
This is way better. Thanks.